build.psm1

```diff
@@ -197,7 +197,7 @@ function Start-PSBuild {
         $Arguments += "build"
     }
     if ($Output) {
-        $Arguments += "--output", (Join-Path $PSScriptRoot $Output)
+        $Arguments += "--output", $Output
```

Comment:
If the value of `$Output` is not an absolute path, will that fail `Start-PSBuild`?
We could normalize the path with `$PSCmdlet.GetUnresolvedProviderPathFromPSPath`.